### DISCUSSION TITLE
Can I use the remix middleware on a non root base path?

### DISCUSSION THREAD

**Discussion by afmelsaidy** [sergiodxa/remix-hono#48] [Q&A] [answered]
Can I mount the remix app on non-root base path?

```
server.use(
	"/dashboard/*",
	remix({ ... }),
);
```

**Comment by sergiodxa** [comment, marked as answer]
You can, but your Remix route needs start all paths with `/dashboard`, otherwise the app will break because it doesn't know what route to use.
